__tests__/tabs.test.js

```diff
@@ -21,7 +21,12 @@ describe('Component page', () => {
 
     await goTo(page, '/components/back-link/')
     await setup(page)
-  })
+
+    // Wait for iframe resizer + 32ms page size setInterval
+    // https://github.com/davidjbradshaw/iframe-resizer/tree/V3#interval
+    await page.waitForSelector('[data-module="app-example-frame"][style]')
+    await new Promise((resolve) => setTimeout(resolve, 32))
```

Comment:
Older releases of `iframe-resizer` poll for changes which appears to hold up the main thread for at least 32ms. This annoying workaround ensures the page is ready for other component handlers to fire in time for tests